stubs/netaddr/netaddr/eui/__init__.pyi

```diff
@@ -0,0 +1,81 @@
+from _typeshed import Self
+from typing import SupportsInt, overload
+from typing_extensions import Literal, SupportsIndex
+
+from netaddr.core import DictDotLookup
+from netaddr.ip import IPAddress
+from netaddr.strategy.eui48 import mac_eui48
+from netaddr.strategy.eui64 import eui64_base
+
+class BaseIdentifier:
+    def __init__(self) -> None: ...
+    def __int__(self) -> int: ...
+    def __long__(self) -> int: ...
+    def __oct__(self) -> str: ...
+    def __hex__(self) -> str: ...
+    def __index__(self) -> int: ...
+    def __eq__(self, other: object) -> bool: ...
+    def __ne__(self, other: object) -> bool: ...
```

Comment:
Where are these defined? I can't see them in the source code

Reply:
These should be defined on the subclasses, I'll move them